apps/studio/components/layouts/ProjectLayout/ProjectLayout.tsx

```diff
@@ -147,33 +143,14 @@ const ProjectLayout = forwardRef<HTMLDivElement, PropsWithChildren<ProjectLayout
             </title>
             <meta name="description" content="Supabase Studio" />
           </Head>
-
-          <div className="flex flex-col h-screen w-screen">
-            {/* Top Nav to access products from mobile */}
-            {!hideIconBar && <MobileNavigationBar />}
-            {!hideHeader && IS_PLATFORM && (
-              <LayoutHeader showHomeLink showProductMenu={!!(showProductMenu && productMenu)} />
-            )}
-            <div className="flex h-full flex-row grow overflow-y-auto">
-              {showProductMenu && productMenu && !(!hideHeader && IS_PLATFORM) && (
-                <MobileViewNav title={product} />
-              )}
-              {/* Left-most navigation side bar to access products */}
-              {!hideIconBar && <NavigationBar />}
-              {/* Product menu bar */}
-              <ResizablePanelGroup
-                className="flex h-full"
-                direction="horizontal"
-                autoSaveId="project-layout"
-              >
-                {/* Existing desktop menu */}
+          <div className="flex flex-row h-full w-full">
+            <div className={cn('w-full overflow-hidden flex-row')}>
+              <ResizablePanelGroup className="" direction="horizontal" autoSaveId="project-layout">
                 {showProductMenu && productMenu && (
                   <ResizablePanel
                     order={1}
-                    defaultSize={1}
                     id="panel-left"
                     className={cn(
-                      'hidden md:flex',
                       'transition-all duration-[120ms]',
                       sideBarIsOpen
                         ? resizableSidebar
```

Comment:
The resizing control is really bad on Firefox, the handle lags a lot. Set `resizableSidebar=false` for both editors, let's get this PR in, we'll figure it out in a followup.

Reply:
there was an issue on firefox that it bugged out if there's no maxWidth prop in.
fixed that.

I won't set resizable to false though since this is a problem already in prod